From 145b6ee3ebf4ff4f0735370473c44287248128a8 Mon Sep 17 00:00:00 2001
From: Kimpe Andy <andykimpe@gmail.com>
Date: Tue, 27 Feb 2024 12:27:12 +0100
Subject: [PATCH] Update macros.scl

---
 rpm/macros.scl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/macros.scl b/rpm/macros.scl
index f1ee5f2..ed5a297 100644
--- a/rpm/macros.scl
+++ b/rpm/macros.scl
@@ -45,7 +45,7 @@ package or when debugging this package.
 %global scl_runtime		%{scl}-runtime
 %global scl_basedir		/opt
 %{!?scl_vendor:			%global scl_vendor rh}
-%{!?_scl_prefix:		%global _scl_prefix %{scl_basedir}/%{scl_vendor}}
+%{!?_scl_prefix:		%global _scl_prefix /opt}
 %global _scl_scripts		%{_scl_prefix}/%{scl}
 %global _scl_root		%{_scl_prefix}/%{scl}/root
 %global _root_prefix		%{_prefix}
