From 5df435de29c11ffa2ec5596f37163b7c35aadf3c Mon Sep 17 00:00:00 2001
From: Kimpe Andy <andykimpe@gmail.com>
Date: Thu, 29 Feb 2024 03:56:43 +0100
Subject: [PATCH] Update scllib.c

---
 src/scllib.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/scllib.c b/src/scllib.c
index 2ba8df8..43a38d8 100644
--- a/src/scllib.c
+++ b/src/scllib.c
@@ -393,7 +393,7 @@ scl_rc list_packages_in_collection(const char *colname, char ***_pkgnames)
     xasprintf(&provide, "scl-package(%s)", colname);
 
     ts = rpmtsCreate();
-    mi = rpmtsInitIterator(ts, RPMDBI_PROVIDENAME, provide, 0);
+    mi = rpmtsInitIterator(ts, RPMTAG_PROVIDENAME, provide, 0);
     while ((h = rpmdbNextIterator(mi)) != NULL) {
 
         srpms[srpms_count++] = headerGetAsString(h, RPMTAG_SOURCERPM);
@@ -575,7 +575,7 @@ static scl_rc owned_by_package(const char *file_path, bool *_owned)
     }
 
     ts = rpmtsCreate();
-    mi = rpmtsInitIterator(ts, RPMDBI_INSTFILENAMES, file_path, 0);
+    mi = rpmtsInitIterator(ts, RPMTAG_FILENAMES, file_path, 0);
 
     *_owned = rpmdbGetIteratorCount(mi) > 0;
 
